modules/ROOT/pages/fd-tour.adoc

```diff
@@ -35,10 +35,10 @@ What are data types?:: Data types show the formats in which data are used in a f
 What are dependencies?:: A dependency is any connector, core Mule Runtime component, API, or module that you use in a flow. The first time that you add one of those objects to a flow, you add the current version of it. Every time afterward that you use that object again in the flow, you use the same version of it. From time to time, that object might be updated in Exchange. If you would like your flow to benefit from the update, you can update the version of the object that you are using in your flow.
 . The toolbar
 +
-Here you can undo and redo changes in a project, download a project, and change the zoom level of the canvas.
-. The control bar
-+
-Here you can run the application for testing, as well as deploy the application to another environment.
+Here you can undo and redo changes in a project, download a project, change the zoom level of the canvas, run the application for testing, as well as deploy the application to another environment.
```

Comment:
"test the application" instead of "run the application for testing"

Reply:
I'd prefer "run the application to test it".
Change "as well as" to "or". "As well as" never ends a list of items.

Reply:
Fixed.